zkevm-circuits/src/evm_circuit/execution/error_oog_call.rs

```diff
@@ -266,11 +276,25 @@ mod test {
 
     fn caller(opcode: OpcodeId, stack: Stack) -> Account {
         let bytecode = call_bytecode(opcode, Address::repeat_byte(0xff), stack);
-        Account::mock_100_ether(bytecode)
+
+        Account {
+            address: Address::repeat_byte(0xfe),
+            balance: Word::from(10).pow(20.into()),
+            code: bytecode.code().into(),
+            ..Default::default()
+        }
     }
 
     fn callee(code: Bytecode) -> Account {
-        Account::mock_code_balance(code)
+        let code = code.code();
+        let is_empty = code.is_empty();
+        Account {
+            address: Address::repeat_byte(0xff),
+            code: code.into(),
+            nonce: if is_empty { 0 } else { 1 }.into(),
+            balance: if is_empty { 0 } else { 0xdeadbeefu64 }.into(),
+            ..Default::default()
+        }
```

Comment:
The mock accounts were refactored away in #1529 to reduce verbosity.
Can we keep both `Account::mock_100_ether(bytecode)` and `Account::mock_code_balance(code)`?

Reply:
updated in [2236916](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1564/commits/22369163147c2f20c622f8521a59b1ef24e13352) thanks !